### PR TITLE
Active record

### PR DESCRIPTION
**Pull request by zilton7**
ActiveRecord:
**Models**
- Add User model with name, email and password columns
- Add Post model with title, body and user references
- Add Comment model with body, user references and post references

**Relations**

User:
- has_many :posts
- has_many :comments

Post:
- has_many :comments
- belongs_to :user

Comment:
- belongs_to :user
- belongs_to :post

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,83 +1,47 @@
...
+# MICRO-REDDIT
 
-## Set-up GitHub Actions
+We were asked to create Micro-Reddit project in order to learn about Active Record. We created User, Post and Comment models, with relationships to each other.
 
-This GitHub Action is going to run [Rubocop](https://docs.rubocop.org/en/stable/) and [Stylelint](https://stylelint.io/) to help you find style issues.
+## Built With
+- Ruby on Rails
+- Active Record
+- Ruby
```
Comment (whiz25) [inline README.md:11]: - Add a section for getting started. List all the commands to run for setting up the project locally

File: app/models/user.rb [hunk 2]
```diff
@@ -0,0 +1,4 @@
+class User < ApplicationRecord
+  has_many :posts
```
Comment (whiz25) [inline app/models/user.rb:3]: - Add validation for user's name

File: app/models/post.rb [hunk 3]
```diff
@@ -0,0 +1,4 @@
+class Post < ApplicationRecord
```
Comment (whiz25) [inline app/models/post.rb:1]: - Add validation for body and title

File: app/models/comment.rb [hunk 4]
```diff
@@ -0,0 +1,4 @@
+class Comment < ApplicationRecord
+  belongs_to :post
```
Comment (whiz25) [inline app/models/comment.rb:3]: - Add validation for the body field

**Review by whiz25 — CHANGES_REQUESTED** [review]
## Changes Required 🙏🏿 
Nice work on this project. You started well. 
Let's make some changes before we can merge this PR.
- [x] I could not create a user, post, or comment. Your routes.rb file is not configured with the necessary routes. Please fix that.
- [x] Check my comments for other suggestions.


### Martin Afani
#### Slack: whiz25

File: app/models/user.rb (hunk 2 above)
Comment (garciajordy) [reply app/models/user.rb:3]: Add validation for User model

File: app/models/post.rb (hunk 3 above)
Comment (garciajordy) [reply app/models/post.rb:1]: Add validation for Post model

File: app/models/comment.rb (hunk 4 above)
Comment (garciajordy) [reply app/models/comment.rb:3]: Add validation for Comment model

File: README.md (hunk 1 above)
Comment (zilton7) [reply README.md:11]: Added full instruction as per request

**Review by descholar-ceo — APPROVED** [review]
# Status: Approved :heavy_check_mark: :heavy_check_mark: :heavy_check_mark:  
## :clap: :clap: :clap: Congratulations @zilton7 and @garciajordy  , your project is approved :tada: :tada: :tada: 
You have implemented all minimum requirements for this project to mark it as complete, it doesn't need more changes, you can go ahead and merge all of the changes on base branch,

Don't forget to fill the mandatory project completion form, and see you to next project,

Happy coding :computer: :computer: :computer: 

Re-reviewed by [Mugirase Emmanuel (descholar)](https://github.com/descholar-ceo)

**Pull request merged**